### PR TITLE
Style fixes

### PR DESCRIPTION
**Pull request by manojava-gk**
Min Height set in notification section to prevent displace of footer on each load

### CODE REVIEW

File: cx-portal/src/components/pages/NotificationCenter/index.tsx [hunk 1]
```diff
@@ -76,6 +76,7 @@ const NotificationGroup = ({
 
 export default function NotificationCenter() {
   const { t } = useTranslation('notification')
+  const sectionElement: any = useRef()
```
Comment (nidhigarg-bmw) [inline cx-portal/src/components/pages/NotificationCenter/index.tsx:79]: will this be any type ?

File: cx-portal/src/components/pages/NotificationCenter/index.tsx [hunk 2]
```diff
@@ -253,14 +260,21 @@ export default function NotificationCenter() {
...
+          <Box
+            ref={sectionElement}
+            sx={{
+              minHeight: height,
+            }}
+          >
+            <ul>
+              {groups &&
+                Object.entries(groups).map((entry: any) => (
+                  <li key={entry[0]}>
```
Comment (nidhigarg-bmw) [inline cx-portal/src/components/pages/NotificationCenter/index.tsx:272]: why key to first index only ?
And entry of any type ?

File: cx-portal/src/components/pages/NotificationCenter/index.tsx (hunk 1 above)
Comment (manojava-gk) [reply cx-portal/src/components/pages/NotificationCenter/index.tsx:79]: yes. No specific type

File: cx-portal/src/components/pages/NotificationCenter/index.tsx (hunk 2 above)
Comment (manojava-gk) [reply cx-portal/src/components/pages/NotificationCenter/index.tsx:272]: Key is first index of each entry.

Comment (manojava-gk) [reply cx-portal/src/components/pages/NotificationCenter/index.tsx:272]: entry is an array here. I do not see any specific reason to define a type for this

**Review by nidhigarg-bmw — APPROVED** [review]

**Pull request merged**